### PR TITLE
OSDOCS-5480: Update welcome page 4.13

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13 only

Issue:
https://issues.redhat.com/browse/OSDOCS-5480

Link to docs preview:
https://57001--docspreview.netlify.app/microshift/latest/welcome/index.html

QE review:
N/A internal docs work
SME approval given

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@dhellmann and @DanielFroehlich PTAL! I added links to RHEL for Edge and the RHDE splash page and dropped in our TP lingo.

File: welcome/index.adoc [hunk 1]
```diff
@@ -62,12 +74,13 @@ Start with **xref:../architecture/architecture.adoc#architecture-overview-archit
 endif::[]
 
 ifdef::microshift[]
-Start with xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}] and xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing].
-Then, see the xref:../microshift_release_notes/microshift-4-13-release-notes.adoc#microshift-4-13-release-notes[release notes].
-
 [IMPORTANT]
 ====
-Red Hat does not provide or support an update or upgrade path from the Technology Preview version to later versions of {product-title}. A new installation is necessary.
+{product-title} is Technology Preview only. This Technology Preview software is not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using {product-title} in production. Technology Preview provides early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```
Comment (mjpytlak) [inline welcome/index.adoc:79]: There is a TP snippet. I suggest using the snippet. In the event our language around TP changes, you will pick up the update.

**Review by mjpytlak — COMMENTED** [review]
@ShaunaDiaz Few items to consider. Thank you.

File: welcome/index.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply welcome/index.adoc:79]: Unfortunately it's not quite accurate for MicroShift. This is most of it with some tweaks.

File: welcome/index.adoc [hunk 2]
```diff
@@ -43,9 +50,14 @@ For documentation that is not specific to {product-title}, see the link:https://
...
 
-For documentation that is not specific to {product-title}, see the link:https://docs.openshift.com/container-platform/latest/welcome/index.html[OpenShift Container Platform documentation].
+Start with xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}] and xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing].
+Then, see the xref:../microshift_release_notes/microshift-4-13-release-notes.adoc#microshift-4-13-release-notes[release notes].
+
+* For information about Red Hat Device Edge, read the link:https://access.redhat.com/documentation/en-us/red_hat_device_edge/4/html/overview/device-edge-overview[Red Hat Device Edge overview].
+* For information about Red Hat Enterprise Linux for Edge, read link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images/index[RHEL for Edge documentation].
+* For documentation that is not specific to {product-title}, read link:https://docs.openshift.com/container-platform/latest/welcome/index.html[OpenShift Container Platform documentation].
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply welcome/index.adoc:60]: We are more current than OCP with inclusive language and don't use "see."

[the remark this answers is not in the document]
Comment (sheriff-rh) [reply welcome/index.adoc:60]: Thanks for the reminder about the word "see". We should probably do a sweep for that at some point.

Can we change it to "read the" <link> on line 59 and 60 to be consistent with line 58?

**Review by sheriff-rh — COMMENTED** [review]
One small request!

File: welcome/index.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply welcome/index.adoc:60]: @sheriff-rh I've had a peer review comment in the past to remove "the" from before link references to whole guides. Do you happen to know where the guidance is? I haven't found this tidbit in our style guides. I personally prefer "the," but perhaps without is better for translations?

The rule _seems to be_ that we only use "the" with titles that aren't full guides, such as here: Start with [Architecture](https://docs.openshift.com/container-platform/4.12/architecture/architecture.html#architecture-overview-architecture) and [Security and compliance](https://docs.openshift.com/container-platform/4.12/security/container_security/security-understanding.html#understanding-security). Then, see the [release notes](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-release-notes).

**Comment by sheriff-rh** [comment]
Discussed changes with Shauna, we are okay to merge.

**Pull request merged**